core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -187,15 +258,27 @@ enum class PlanarFace(val int: Int) {
 	val adjacencies: Array<PlanarFace> get() = PlanarFace.ADJACENCIES[int % 3]
 }
 
+/**
+ * A generic that represents the location of something and its neighbors.
+ */
 interface Locator {
 	val vec3i: Vec3i
+
+	/**
+	 * A list of neighbors in which connections are possible with.
+	 */
 	fun neighbors(): List<Locator>
 
 	/* When overriding this, put more specific tests in more specific locators, as a rule */
+	/**
+	 * Returns true if it is possible for this node and another node to connect.
```

Comment:
Here, again, the comment above should probably be integrated, but in this case it is poorly phrased on its own.

What I meant to say was something more like: "as a rule, when overriding this, put the tests for compatibility with less specific Locators into more specific Locators' implementations."